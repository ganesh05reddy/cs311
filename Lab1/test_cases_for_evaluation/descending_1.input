	.data
a:
	70
	80
	40
	20
n:
	4
