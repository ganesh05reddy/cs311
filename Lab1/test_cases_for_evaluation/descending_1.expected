Main Memory Contents:

0		: 80
1		: 70
2		: 40
3		: 20
